data/events/2022-los-angeles.yml

```
@@ -142,6 +142,8 @@ sponsors:
     level: silver
   - id: sonatype
     level: gold
+  - id: stormforge
+    level: startup
```

You'll either need to use "bronze" for this level, or change the ID of the startup level (you have id "bronze" labelled as "startup", so you'll need to match the IDs for this to show up. 
```suggestion
    level: bronze
```